full_upgrade/for_mk2_mk2s/manual/assembly_instructions/step01.md

```diff
@@ -17,8 +17,9 @@
 1. If you did not get your 90° joining plates from RatRig, file sharp edges on the diagonal of 90° joining plates (figure 1.1). A small chamfer is enough, this just to avoid scratching cables and it will be useful for the next step as well
 1. Assemble 90° joining plates to the two 311mm v-slots as seen on figure 1.2 (with 8x M5x10 screws and 8x tee nuts). Do not tighten M5 screws yet
 1. Add 1x 370mm v-slot vertically as seen on figure 1.3 (with 4x M5x10 screws and 4x tee nuts). Do not tighten M5 screws yet
-1. Add 3 tee nuts on each side of the 311mm v-slots (12x tee nuts in total) as seen on figure 1.4
-
+1. Add 3 tee nuts on each side of the 311mm v-slots (12x tee nuts in total) as seen on figure 1.4 <br>
+   :warning: this step is very important, double check you have use 12 Tee nuts (3 on each side of each 311 V-Slots as seen       on figure 1.4). If you forget this step, you will have to restart build from scratch ! If it's still not clear, have a look at this [picture](https://github.com/gregsaun/prusa_i3_bear_upgrade/blob/master/full_upgrade/for_mk2_mk2s/manual/assembly_instructions/img/fig6.4.jpg) to understand what it is for.  

```

Comment:
The link needs updating to point at the correct image:
https://github.com/gregsaun/prusa_i3_bear_upgrade/blob/master/full_upgrade/for_mk2_mk2s/manual/assembly_instructions/img/fig7.1.jpg works well.